app/src/main/java/com/firebase/uidemo/auth/LeakCatcher.java

```diff
@@ -2,13 +2,18 @@
 
 import android.app.Application;
 import android.content.Context;
+import android.support.v7.app.AppCompatDelegate;
 
 import com.squareup.leakcanary.LeakCanary;
 import com.squareup.leakcanary.RefWatcher;
 
 public class LeakCatcher extends Application {
     private RefWatcher mRefWatcher;
 
+    static {
+        AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_AUTO);
```

Comment:
Feels a little strange to do this in a class called 'LeakCatcher'.  Also should we use 'MODE_NIGHT_SYSTEM' to respect the system setting?